src/migrate/azext_migrate/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67",
+    "azext.maxCliCoreVersion": "3.0.0"

```

Comment:
not necessary

Reply:
This should have been fixed in the latest code generator